pom.xml

```diff
@@ -378,6 +378,7 @@
                         <systemPropertyVariables>
                            <!-- See test org.hibernate.ogm.test.utils.EnvironmentTest -->
                            <hibernate.service.allow_crawling>false</hibernate.service.allow_crawling>
+                           <user.timezone>UTC</user.timezone>
```

Comment:
Why do you need this? Also, this is an example of a change that should be in a separate commit